index.js

```diff
@@ -17,13 +17,31 @@
 
 'use strict';
 
-var forEach = require('for-each');
-var warning = require('warning');
-var has = require('has');
-var trim = require('string.prototype.trim');
+/* Should always be false */
+var debug = false;
+
+var forEach = function (iterable, callback) {
+  var keys = Object.keys(iterable);
+  for (var i = 0; i < keys.length; i += 1) {
+    var key = keys[i];
+    var val = iterable[key];
+    callback(val, key, iterable);
+  }
+};
+
+var has = function (object, property) {
+  return Object.prototype.hasOwnProperty.call(object, property);
```

Comment:
this exposes the package to breakage from `delete Object.prototype.hasOwnProperty`, for example, that the `has` package protects against.